packages/styles/button.css

```
@@ -81,7 +80,7 @@ button.Link {
 .Button--primary:focus:before,
 .Button--secondary:focus:before,
 .Button--error:focus:before {
-  box-shadow: 0 0 1px 2px var(--button-focus-ring-color);
+  box-shadow: 0 0 1px 2px var(--focus);
```

I think what we were going for is to allow a consumer to locally override the focus ring just for button, I still think we can do this be providing a fallback:

```suggestion
  box-shadow: 0 0 1px 2px var(--button-focus-ring-color, --focus);
```

This should allow us to maintain backwards compat - and probably something we need to make a note of in the future.